README.md

```diff
@@ -1,3 +1,31 @@
 # FusionAuth Example React App With SDK
-This is an example react app made with [https://github.com/FusionAuth/fusionauth-react-sdk](FusionAuth's React SDK)
-Currently under development.
\ No newline at end of file
+This is an example react app made with [FusionAuth's React SDK](https://github.com/FusionAuth/fusionauth-react-sdk).
+
+## Prerequisites
+You will need to have the following things installed on your computer:
+ - [Git](https://git-scm.com/): This is will used to `git clone` the repo.
+ - [NodeJS](https://nodejs.org/en/): This will be used in order to run the node server, and also includes `npm` which will be needed for pulling down the various dependencies.
+
+## Installation
+Run the following commands inside of a terminal
+ - `git clone https://github.com/FusionAuth/fusionauth-example-react-with-sdk.git`
+ - `cd fusionauth-example-react-with-sdk`: This is the root directory for the example app.
+ - `cd client && npm install`: This will install the dependencies for the React application.
+ - `cd ../server && npm install`: This will install the dependencies for the Express application.
```

Comment:
We should be using `yarn` not NPM since we have committed the yarn lock file
```suggestion
 - `cd client && yarn install`: This will install the dependencies for the React application.
 - `cd ../server && yarn install`: This will install the dependencies for the Express application.
```